### DISCUSSION TITLE
Possible to record both hi & low res streams?

### DISCUSSION THREAD

**Discussion by ghgeiger** [blakeblackshear/frigate#3928] [Ask A Question] [answered]
I currently have frigate setup to record 5 low res h.264 substreams whenever an object is detected. I also have a separate h.265 NVR that records high res substreams from all cameras 24/7. I would like to drop the dedicated NVR and have frigate continue to work as it currently does, but I would like it to also keep a copy of my h.265 streams, ideally on any motion event (not just those recognized by ai), but I'd also be ok with either 24/7 or just on recognized motion events. Is there any way to set this up? I did attempt to set this up on 0.10.1 by setting up a -record setting in each camera's config with the high res stream, however for some reason, frigate then failed to use the low res clips for playback on the website/home assistant & that didn't work very well with h.265. I guess what I'm getting at is I'd like to know how to setup frigate so that it keeps a high res archive of all motion events while using low res for recognition & playback clips on web & app.

**Comment by NickM-27** [comment, marked as answer]
It is not currently possible to record on multiple streams unless you define them as separate cameras. There is an open feature request to do this.

**Comment by ghgeiger** [comment]
Thanks for the response. I've added a comment on the issue that I think you're referring to so I can follow its progress.

**Reply by NickM-27** [reply]
yes that was it
